documentation/modules/security/proc-installing-certs-per-listener.adoc

```diff
@@ -53,7 +53,7 @@ listeners:
 # ...
 ----
 +
-.Example configuration for a TLS listener
+.Example configuration for an mTLS listener
```

Comment:
I think this is confusing. Both here and in the example above. You focus about the custom certificate. But that certificate is not used for authentication but only as an server certificate. So I think haveing the mTLS authentication is misleading here and we should both here and above remove the 
```
authentication:
  type: tls
```
section completely and not refer to any mTLS.